### ISSUE TITLE
Issues about flashed the odx on my 2022 Audi A4L:coded actual installation not detected

### ISSUE THREAD

**Issue by ousttt** [jilleb/mqb-soundaktor#39]
Regarding the problem of my 2022 A4L, I flashed the  odx according to the tutorial, but he showed failure, and then my soundaktor showed on the software of odis14.1 version: coded actual installation not detected. I see a question mark in front of the soundaktor in the gateway, and I can't encode it. I hope you can help me, my soundaktor is basically in a state of no effect now.thx!

**Comment by jilleb** [comment]
Are you sure you flashed a xxx907159 firmware on the  right hardware?
Is it still not working after reboot of the car?

**Comment by ousttt** [comment]
yep I installed the right firmware. I reboot the car and the soundaktor didn't work anymore.

发自我的iPhone


------------------ Original ------------------
From: Chillout ***@***.***&gt;
Date: Thu,Aug 24,2023 3:16 PM
To: jilleb/mqb-soundaktor ***@***.***&gt;
Cc: ousttt ***@***.***&gt;, Manual ***@***.***&gt;
Subject: Re: [jilleb/mqb-soundaktor] Issues about flashed the odx on my 2022Audi A4L:coded actual installation not detected (Issue #39)





 
Are you sure you flashed a xxx907159 firmware on the  right hardware?
 Is it still not working after reboot of the car?
